src/overlays/actors/ovl_En_Bigslime/z_en_bigslime.c

```diff
@@ -48,289 +243,2911 @@ const ActorInit En_Bigslime_InitVars = {
+    { 126, 0.65f }, { 132, 0.25f }, { 138, 0.3f }, { 144, 0.15f }, { 150, 0.45f }, { 156, 0.3f },  { 161, 0.25f },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigslime/EnBigslime_Draw.s")
+void EnBigslime_DrawBigslime(Actor* thisx, GlobalContext* globalCtx) {
+    EnBigslime* this = THIS;
+    EnBigslimeBubbles* bubbles;
+    Vtx* dynamicVtx;
+    MtxF* sysMatrix;
+    s32 i;
+
+    func_8012C2DC(globalCtx->state.gfxCtx);
+    func_800B8118(&this->actor, globalCtx, 0);
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+
+    gSPSegment(POLY_XLU_DISP++, 0x09, sBigslimeDynamicVtx[this->dynamicVtxState]);
+    gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+    gSPDisplayList(POLY_XLU_DISP++, &D_06010530);
+    gSPDisplayList(POLY_XLU_DISP++, &D_060105E8);
+
+    if ((this->actionFunc == EnBigslime_Freeze) || (this->actionFunc == EnBigslime_FrozenGround) ||
+        (this->actionFunc == EnBigslime_FrozenFall) || (this->actionFunc == EnBigslime_Melt)) {
+        AnimatedMat_Draw(globalCtx, this->bigslimeFrozenTex);
+        gSPSegment(POLY_XLU_DISP++, 0x09, sBigslimeTargetVtx);
+        gSPDisplayList(POLY_XLU_DISP++, &D_06010B80);
+        gSPDisplayList(POLY_XLU_DISP++, &D_060105E8);
+    }
+
+    if (this->actor.scale.x > 0.0f) {
+        Matrix_SetCurrentState(&globalCtx->mf_187FC);
+        Matrix_Scale(0.0050000003f, 0.0050000003f, 0.0050000003f, MTXMODE_APPLY);
+        sysMatrix = Matrix_GetCurrentState();
+
+        for (i = 0; i < 28; i++) {
+            bubbles = &sBubbles[i];
+            dynamicVtx = &sBigslimeDynamicVtx[this->dynamicVtxState][bubbles->v];
+            sysMatrix->wx = dynamicVtx->n.ob[0] * this->actor.scale.x * bubbles->scaleVtx + this->actor.world.pos.x;
+            sysMatrix->wy = dynamicVtx->n.ob[1] * this->actor.scale.y * bubbles->scaleVtx + this->actor.world.pos.y;
+            sysMatrix->wz = dynamicVtx->n.ob[2] * this->actor.scale.z * bubbles->scaleVtx + this->actor.world.pos.z;
+            gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx),
+                      G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            gSPDisplayList(POLY_XLU_DISP++, &D_0600F9D0);
+        }
+    }
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+
+    EnBigslime_SetSysMatrix(&this->actor.world.pos, globalCtx, D_06011050, this->vtxScaleX, this->vtxScaleZ,
+                            this->actor.scale.y * BIGSLIME_RADIUS_F, this->rotation, 175.0f);
+    EnBigslime_DrawGekko(&this->actor, globalCtx);
+}
+
+/* index -1: Limb Not used
+ * index 0:  GEKKO_LIMB_WAIST
+ * index 1:  GEKKO_LIMB_L_SHIN
+ * index 2:  GEKKO_LIMB_L_FOOT
+ * index 3:  GEKKO_LIMB_R_SHIN
+ * index 4:  GEKKO_LIMB_R_FOOT
+ * index 5:  GEKKO_LIMB_L_UPPER_ARM
+ * index 6:  GEKKO_LIMB_L_FOREARM
+ * index 7:  GEKKO_LIMB_L_HAND
+ * index 8:  GEKKO_LIMB_R_UPPER_ARM
+ * index 9:  GEKKO_LIMB_R_FOREARM
+ * index 10: GEKKO_LIMB_R_HAND
+ * index 11: GEKKO_LIMB_JAW
+ */
```

Comment:
Call these elements instead of index?

Reply:
I might use an even simpler term: `value`, since element can have a separate meaning in the context of vertices and triangles, which this file has a lot of